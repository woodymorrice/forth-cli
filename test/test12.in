true while false dup while true dup not loop = loop
. . .