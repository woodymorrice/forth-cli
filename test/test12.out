binary ops require at least 2 values on the stack
